src/auth/constants.ts

```diff
@@ -43,19 +23,8 @@ const loginProviderTunnistusProperties: LoginProviderProps = {
   sessionPollerSettings: { pollIntervalInMs: 300000 }, // 300000ms = 5min
 };
 
-export const oidcProviderName: OidcProviderName =
-  import.meta.env.VITE_OIDC_PROVIDER || 'tunnistus';
-// By default use Tunnistus SSO
-export const loginProviderProperties =
-  oidcProviderName === 'tunnistamo'
-    ? loginProviderTunnistamoProperties
-    : loginProviderTunnistusProperties;
-const tunnistamoApiTokenKeyName: string =
-  import.meta.env.VITE_OPENID_CONNECT_API_TOKEN_KEY ||
-  'https://api.hel.fi/auth/mvj';
+// Use Tunnistus SSO
+export const loginProviderProperties = loginProviderTunnistusProperties;
 const tunnistusApiTokenKeyName: string =
   import.meta.env.VITE_TUNNISTUS_OIDC_API_AUDIENCE || 'mvj-api';
-export const apiTokenKeyName =
-  oidcProviderName === 'tunnistamo'
-    ? tunnistamoApiTokenKeyName
-    : tunnistusApiTokenKeyName;
+export const apiTokenKeyName = tunnistusApiTokenKeyName;
```

Comment:
We don't need the intermediate constants `loginProviderTunnistusProperties` and `tunnistusApiTokenKeyName`, because we only have one provider. But that is a matter of taste.

Reply:
Correct, I left it as is because it is a bit more detailed. Slightly easier in the future to provide different options based on some condition too.